src/os/user/iterate_plan9.go

```diff
@@ -0,0 +1,101 @@
+package user
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"strings"
+)
+
+// users and groups file location in plan9
+var (
+	usersFile = "/adm/users"
+)
```

Comment:
- single-variables don't need a block.
- since this is unconventional (should be a const), explain that this is mutated in tests.